From 21ba68ae7991ee0f6546d765dcad2c84cd05a375 Mon Sep 17 00:00:00 2001
From: Stephan Bergmann <sbergman@redhat.com>
Date: Sat, 20 Oct 2018 14:38:35 +0200
Subject: Java 11 no longer synthesizes DocumentView$1.class

...so, for simplicity, just include whatever generated DocumentView$*.class by
wildcard

Change-Id: I779e2709c8ef2859d68233300302dd62dbe2455f
Reviewed-on: https://gerrit.libreoffice.org/62073
Tested-by: Jenkins
Reviewed-by: Stephan Bergmann <sbergman@redhat.com>
---
 odk/examples/DevelopersGuide/OfficeDev/DesktopEnvironment/Makefile | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/odk/examples/DevelopersGuide/OfficeDev/DesktopEnvironment/Makefile b/odk/examples/DevelopersGuide/OfficeDev/DesktopEnvironment/Makefile
index 4227108..2048ade 100644
--- a/odk/examples/DevelopersGuide/OfficeDev/DesktopEnvironment/Makefile
+++ b/odk/examples/DevelopersGuide/OfficeDev/DesktopEnvironment/Makefile
@@ -66,8 +66,7 @@ APP1_JAVAFILES  = \
 APP1_CLASSFILES = $(patsubst %.java,$(OUT_APP_CLASS)/%.class,$(APP1_JAVAFILES))
 APP1_CLASSNAMES = $(patsubst %.java,%.class,$(APP1_JAVAFILES)) \
 		CustomizeView$(QUOTE)$$ClickListener.class \
-		DocumentView$(QUOTE)$$1.class \
-		DocumentView$(QUOTE)$$Reactor.class
+		DocumentView$(QUOTE)$$*.class
 
 
 SDK_CLASSPATH = $(subst $(EMPTYSTRING) $(PATH_SEPARATOR),$(PATH_SEPARATOR),$(CLASSPATH)\
-- 
cgit v1.1

